listeners.py

```diff
@@ -33,13 +34,19 @@ def process_timeout(self, delta_seconds):
 class StatisticsWatcher(BaseWatcher):
     def __init__(self, get_logfile):
         self.stats = dict()
+        self.rss_results = sampler.rss_summary()
```

Comment:
Nit: It is better to avoid usage of different terms for the same thing. `rss_summary` is good as for the method as well as for the variable.

Reply:
Sure, changed rss_results -> rss_summary.